include/functions.h

```diff
@@ -896,7 +896,7 @@ void func_800BD9A0(void); // func_800BD9A0
 void func_800BD9E0(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE2 param_6); // func_800BD9E0
 void func_800BDAA0(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE2 param_6); // func_800BDAA0
 void func_800BDB6C(void); // func_800BDB6C
-void func_800BDC5C(void); // func_800BDC5C
+void func_800BDC5C(); // func_800BDC5C
```

Comment:
```void func_800BDC5C(void);``` is preferred since it tells the compiler there _can_ be no args to this function.

Reply:
> 
> 
> Small nitpick on a function prototype.
> Otherwise, looks good to me.
> Getting an error while building. I am looking into it to see if the code or my setup is the cause.

The error I get is
```
mips-linux-gnu-ld -r -T linker_scripts/code_script.txt -T undef.txt -T linker_scripts/object_script.txt --no-check-sections --accept-unknown-input-arch -N -o build/code_pre_dmadata.elf
mips-linux-gnu-ld: cannot find build/src/code/z_collision_check.o
make[1]: *** [Makefile:130: build/code_pre_dmadata.elf] Error 1
make[1]: Leaving directory '/mnt/d/Zelda/mm'
make: *** [Makefile:109: all] Error 2
```